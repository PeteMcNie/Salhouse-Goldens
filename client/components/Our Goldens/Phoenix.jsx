import React from 'react'

import Image from 'react-bootstrap/Image'

import NavbarComp from '../NavbarComp'

function Phoenix () {
  return (
    <>
      <NavbarComp />
      <div className='belowNavbar'>
        <h2>Phoenix</h2>
      </div>
      <div>
        <h3>Phoenix</h3>
        <p>Sire: Braeview the Kingmaker</p>
        <p>Dam: Honey of Frandale</p>
        <p>10 January 2002 - 14 September 2016</p>
        <div>
          <Image src="images/phoenix1.jpg" fluid />
        </div>
      </div>
      <div>
        <p>
            Phoenix was my first golden and is a beautiful pet. I have dabbled in agility and a tiny bit of obedience with her but
            really she is mainly a pet who lives with us and enjoys herself, mostly. She does have one or two phobias, e.g. she
            doesn&apos;t like thunder and tends to be a bit claustrophobic. But other than that, she is a lovely pet. She is very gentle
            and light on her feet and loves nothing more than to have a tug of war with whoever might be around to play, be it
            human or dog!
        </p>
        <p>
            About three years ago she was diagnosed with cataracts forming in her eyes. Then a year ago we discovered that she also
            had retinal degeneration. She is 8 years old now and is partially blind. We may decide to have the cataracts operated
            on some time this year, but there is no cure for the retinal degeneration which will eventually make her totally blind.
            So it will be interesting to see how she gets on over the next couple of years. At the moment, even though her eyesight
            is generally poor, when we go for walks in the bush, even at night, she is far better finding her way along the path
            than I am. I who have to have a torch and fall off the path if I am not careful. Dogs obviously work on other senses
            than eyesight to guide them. We will have to help her though and it will be a challenge but one that we will face together.
            Phoenix was spayed when she was young so we will not be passing this particular problem on to others.
        </p>
        <p>
            To find out more about how Phoenix came into our lives, go to the <a href='/mylifewithanimals'>My Life with Animals</a> page.
        </p>
        <p>
            Phoenix finally passed away on 14 September 2016. That was a very sad day. No, I can&apos;t just say that... that was a
            devastating day. We did it all in the best way we could for her and for that I am forever thankful. However, it was
            still too soon. I wanted to hold her forever and keep her, but for her sake it was kindest to let her go. She took a
            piece of our hearts with her and we will never forget her. Our lasting message for our beautiful Phoenix was &ldquo;Thank you
            for sharing your life with us.&ldquo; I miss you so very much my beautiful girl! xx
        </p>
      </div>
      <hr />
      <div>
        <p>Photos:</p>
        <Image src="images/phoenix2.jpg" fluid />
        <p>Puppy Phoenix used to be a doorstop!</p>
        <Image src="images/phoenix3.jpg" fluid />
        <p>We are not amused at bath time!</p>
        <Image src="images/phoenix4.jpg" fluid />
        <p>We do, however, love watching fish!</p>
        <Image src="images/phoenix5.jpg" fluid />
        <p>And playing tug of war with anyone - even Shadow!</p>
        <Image src="images/phoenix6.jpg" fluid />
        <Image src="images/phoenix7.jpg" fluid />
        <p>And we love opening Christmas presents!</p>
        <Image src="images/phoenix8.jpg" fluid />
        <p>Phoenix can also do the serious stuff too - 1st prize in elementary agility!</p>
        <Image src="images/phoenix9.jpg" fluid />
        <p>Jumping through the tyre.</p>
        <Image src="images/phoenix10.jpg" fluid />
        <p>Phoenix posing.</p>
        <Image src="images/phoenix11.jpg" fluid />
        <p>Phoenix meets a friend.</p>
        <Image src="images/phoenix12.jpg" fluid />
        <p>Phoenix at The Pinnacles!</p>
        <Image src="images/phoenix13.jpg" fluid />
        <p>Lady muck, posing again!</p>
      </div>
    </>
  )
}

export default Phoenix
