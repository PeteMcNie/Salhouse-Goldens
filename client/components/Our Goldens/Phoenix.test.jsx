import React from 'react'
import { render } from '@testing-library/react'

import Phoenix from './Phoenix'

test('Phoenix component matches snapshot', () => {
  const { container } = render(<Phoenix />)
  expect(container).toMatchSnapshot()
})
