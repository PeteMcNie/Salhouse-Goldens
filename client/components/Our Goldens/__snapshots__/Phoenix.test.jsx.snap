// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Phoenix component matches snapshot 1`] = `
<div>
  <nav
    class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top"
  >
    <a
      class="navbar-brand"
      href="/"
    >
      <img
        alt="Goldens retrievers, phoenix and kula"
        class="shlogo d-inline-block align-top"
        src="images/logo.jpeg"
      />
    </a>
    <button
      aria-controls="responsive-navbar-nav"
      aria-label="Toggle navigation"
      class="navbar-toggler collapsed"
      type="button"
    >
      <span
        class="navbar-toggler-icon"
      />
    </button>
    <div
      class="navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <div
        class="ml-auto navbar-nav"
      >
        <a
          class="nav-link"
          data-rb-event-key="/"
          href="/"
        >
          Home
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Our Goldens
          </a>
        </div>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Grooming Your Golden
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="mylifewithanimals"
          href="mylifewithanimals"
        >
          My Life With Animals
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            About Us
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="#links"
          href="#links"
        >
          Links
        </a>
        <a
          class="nav-link"
          data-rb-event-key="#contact"
          href="#contact"
        >
          Contact
        </a>
      </div>
    </div>
  </nav>
  <div
    class="belowNavbar"
  >
    <h2>
      Phoenix
    </h2>
  </div>
  <div>
    <p>
      Sire: Braeview the Kingmaker
    </p>
    <p>
      Dam: Honey of Frandale
    </p>
    <p>
      10 January 2002 - 14 September 2016
    </p>
    <div>
      <img
        class="img-fluid"
        src="images/phoenix1.jpg"
      />
    </div>
  </div>
  <div>
    <p>
      Phoenix was my first golden and is a beautiful pet. I have dabbled in agility and a tiny bit of obedience with her but really she is mainly a pet who lives with us and enjoys herself, mostly. She does have one or two phobias, e.g. she doesn't like thunder and tends to be a bit claustrophobic. But other than that, she is a lovely pet. She is very gentle and light on her feet and loves nothing more than to have a tug of war with whoever might be around to play, be it human or dog!
    </p>
    <p>
      About three years ago she was diagnosed with cataracts forming in her eyes. Then a year ago we discovered that she also had retinal degeneration. She is 8 years old now and is partially blind. We may decide to have the cataracts operated on some time this year, but there is no cure for the retinal degeneration which will eventually make her totally blind. So it will be interesting to see how she gets on over the next couple of years. At the moment, even though her eyesight is generally poor, when we go for walks in the bush, even at night, she is far better finding her way along the path than I am. I who have to have a torch and fall off the path if I am not careful. Dogs obviously work on other senses than eyesight to guide them. We will have to help her though and it will be a challenge but one that we will face together. Phoenix was spayed when she was young so we will not be passing this particular problem on to others.
    </p>
    <p>
      To find out more about how Phoenix came into our lives, go to the My Life with Animals page.
    </p>
    <p>
      Phoenix finally passed away on 14 September 2016. That was a very sad day. No, I can't just say that... that was a devastating day. We did it all in the best way we could for her and for that I am forever thankful. However, it was still too soon. I wanted to hold her forever and keep her, but for her sake it was kindest to let her go. She took a piece of our hearts with her and we will never forget her. Our lasting message for our beautiful Phoenix was “Thank you for sharing your life with us.“ I miss you so very much my beautiful girl! xx
    </p>
  </div>
  <hr />
  <div>
    <p>
      Photos:
    </p>
    <img
      class="img-fluid"
      src="images/phoenix2.jpg"
    />
    <p>
      Puppy Phoenix used to be a doorstop!
    </p>
    <img
      class="img-fluid"
      src="images/phoenix3.jpg"
    />
    <p>
      We are not amused at bath time!
    </p>
    <img
      class="img-fluid"
      src="images/phoenix4.jpg"
    />
    <p>
      We do, however, love watching fish!
    </p>
    <img
      class="img-fluid"
      src="images/phoenix5.jpg"
    />
    <p>
      And playing tug of war with anyone - even Shadow!
    </p>
    <img
      class="img-fluid"
      src="images/phoenix6.jpg"
    />
    <img
      class="img-fluid"
      src="images/phoenix7.jpg"
    />
    <p>
      And we love opening Christmas presents!
    </p>
    <img
      class="img-fluid"
      src="images/phoenix8.jpg"
    />
    <p>
      Phoenix can also do the serious stuff too - 1st prize in elementary agility!
    </p>
    <img
      class="img-fluid"
      src="images/phoenix9.jpg"
    />
    <p>
      Jumping through the tyre.
    </p>
    <img
      class="img-fluid"
      src="images/phoenix10.jpg"
    />
    <p>
      Phoenix posing.
    </p>
    <img
      class="img-fluid"
      src="images/phoenix11.jpg"
    />
    <p>
      Phoenix meets a friend.
    </p>
    <img
      class="img-fluid"
      src="images/phoenix12.jpg"
    />
    <p>
      Phoenix at The Pinnacles!
    </p>
    <img
      class="img-fluid"
      src="images/phoenix13.jpg"
    />
    <p>
      Lady muck, posing again!
    </p>
  </div>
</div>
`;
